AirCasting/SDSync/ViewsFlow/SDSyncingAB/SDSyncViewModel.swift

```diff
@@ -5,8 +5,12 @@ import Combine
 import CoreBluetooth
 
 struct SDSyncProgressViewModel {
-    let label: String
-    let value: String
+    let sessionType: String
+    let progress: SDCardProgress
+    
+    func toString() -> String {
```

Comment:
won't be needed after changes from previous comment - this will be done inside `SDSyncViewModel`